entries/l/lego.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "LEGO": {
+    "domain": "lego.com",
+    "tfa": [
+      "email"
+    ],
+    "notes": "To turn on, Go to Account Overview > Personal & Address Details > LEGO Account Details > Security > Use two-factor authentication",
+    "categories": [
+      "creativity",
```

Comment:
```suggestion
```

This doesn't fit with the other `creativity` sites

Reply:
I am still trying to figure out the categories so thank you for the feedback here.